common/lib/xmodule/xmodule/tests/test_video.py

```diff
@@ -993,3 +1000,77 @@ def test_video_with_no_transcripts_translation_retrieval(self):
         descriptor = instantiate_descriptor(data=None)
         translations = descriptor.available_translations(descriptor.get_transcripts_info(), verify_assets=False)
         self.assertEqual(translations, ['en'])
+
+    def test_video_with_lang_with_no_transcripts_translation(self):
```

Comment:
I would say to change the name to more readable name, Maybe you can choose `test_video_with_language_do_not_have_transcripts_translation`.